src/types/WebChatConfig.ts

```diff
@@ -0,0 +1,358 @@
+/**
+ * (C) Copyright IBM Corp. 2022.
+ *
```

Comment:
Especially with config,  but also with the service desk types, just sketched out with us providing types for something that is version based. Perhaps the real answer here is to just start doing a 1:1 publish of this package with every version of web chat and provide proper types for each version of web chat.